config/routes.rb

```diff
@@ -55,5 +55,7 @@
   #   end
 
   root 'home#index'
-  resources :cards
+  resources :cards do
+      patch 'trainer/review', on: :member, path: 'review'
```

Comment:
Use 2 (not 4) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.